src/scenes/LoginScene/LoginScene.test.tsx

```diff
@@ -11,3 +13,186 @@ describe('onSubmit', () => {
+    });
+
+    it('should have finnish email placeholder', () => {
+      const word = wrapper.find('[name="email"]').prop('label');
+      expect(word).toBe('Sähköposti');
+    });
+
+    it('should have finnish password placeholder', () => {
+      const word = wrapper.find('[name="password"]').prop('label');
+      expect(word).toBe('Salasana');
+    });
+
+    it('should have finnish button text', () => {
+      const word = wrapper.find('Button').prop('children');
+      expect(word).toBe('Kirjaudu');
+    });
+  });
+
+  describe('English translations', () => {
+    let wrapper: ShallowWrapper;
+    beforeAll(() => {
+      wrapper = shallow<LoginScene>(<LoginScene locale={ReleoxLocale.EN} />);
+    });
+
+    it('should have english title', () => {
+      const word = wrapper.find('AuthLayout').prop('title');
+      expect(word).toBe('Login');
+    });
+
+    it('should have english subtitle', () => {
+      const word = wrapper.find('AuthLayout').prop('subTitle');
+      expect(word).toBe('Fill form and login');
+    });
+
+    it('should have english email placeholder', () => {
+      const word = wrapper.find('[name="email"]').prop('label');
+      expect(word).toBe('Email');
+    });
+
+    it('should have english password placeholder', () => {
+      const word = wrapper.find('[name="password"]').prop('label');
+      expect(word).toBe('Password');
+    });
+
+    it('should have english button text', () => {
+      const word = wrapper.find('Button').prop('children');
+      expect(word).toBe('Login');
+    });
+  });
+
+  describe('getLinks test', () => {
+    it('should have default links', () => {
+      const wrapper = shallow<LoginScene>(<LoginScene />);
+      const links = wrapper.find('AuthLayout').prop('links');
+      expect(links).toEqual([
+        {
+          to: '/forgot',
+          id: 'LoginScene-forgot-link',
+          text: 'Unohditko salasanasi?',
+        }],
+      );
+    });
+
+    it('should have default links in english', () => {
+      const wrapper = shallow<LoginScene>(<LoginScene locale={ReleoxLocale.EN} />);
+      const links = wrapper.find('AuthLayout').prop('links');
+      expect(links).toEqual([
+        {
+          to: '/forgot',
+          id: 'LoginScene-forgot-link',
+          text: 'Forgot password?',
+        }],
+      );
+    });
+
+    it('should have register links also', () => {
+      window.RELEOX_OPTIONS = { showRegisterLink: true };
+      const wrapper = shallow<LoginScene>(<LoginScene />);
+      const links = wrapper.find('AuthLayout').prop('links');
+      expect(links).toEqual([
+        {
+          to: '/forgot',
+          id: 'LoginScene-forgot-link',
+          text: 'Unohditko salasanasi?',
+        },
+        {
+          to: '/register',
+          id: 'LoginScene-register-link',
+          text: 'Oletko uusi? Luo tunnus!',
+        }],
+      );
+    });
+
+    it('should have register links also in english', () => {
```

Comment:
it should show register link in english if showRegisterLink is set to true and locale is set to EN